roles/mig_controller_deploy/defaults/main.yml

```diff
@@ -1,2 +1 @@
 ---
-with_controller: true
```

Comment:
Check on this one, needs to be defined ...

Reply:
Just checked, was never used. https://github.com/Danil-Grigorev/mig-ci/blob/d250f2e5284803b950f5f875446a75bb43716720/roles/mig_controller_deploy/tasks/main.yml#L12 You use `mig_controller_host_cluster`. Going to substitute this one.